### PR TITLE
26 decrosstalk episodic mean fov webm looks weird

### PR DESCRIPTION
**Pull request by arielleleon**
- fixes webm by adding normalized image
- creates max and avg projection images

### CODE REVIEW

File: code/paired_plane_registration.py [hunk 1]
```diff
@@ -434,18 +461,32 @@ def episodic_mean_fov(
         num_frames = min(num_frames, epoch_interval)
         start_frames = [num_frames // 2 + i * epoch_interval for i in range(num_epochs)]
```
Comment (jkim0731) [inline code/paired_plane_registration.py:462]: Lines from 458 to 462 need explanation on why we want to remove about half of each epoch length at the beginning and at the end.

File: code/paired_plane_registration.py [hunk 2]
```diff
@@ -434,18 +461,32 @@ def episodic_mean_fov(
...
+        ],
+    ):
+        plt.imsave(dest, im, cmap="gray")
+
     with h5py.File(save_path, "w") as f:
         f.create_dataset("data", data=mean_fov)
     if save_webm:
-        encode_video(mean_fov, str(webm_path), 3)
+        norm_array = normalize_array(mean_fov)
+        encode_video(norm_array, str(webm_path), 5)
```
Comment (jkim0731) [inline code/paired_plane_registration.py:491]: Any reason for increasing frame rate?

**Review by jkim0731 — COMMENTED** [review]
Was it verified that the normalization makes videos look OK?

**Review by jkim0731 — APPROVED** [review]
Looks good!
